lib/private/legacy/OC_Image.php

```diff
@@ -51,7 +51,7 @@
 class OC_Image implements \OCP\IImage {
 
 	// Default memory limit for images to load (128 MBytes).
```

Comment:
The comment should be adjusted, too.

```suggestion
	// Default memory limit for images to load (256 MBytes).
```

Reply:
fixed